src/Misc.ino

```diff
@@ -306,6 +306,20 @@ bool allocatedOnStack(const void* address) {
 #endif // ESP32
 
 
+uint16_t espeasy_analogRead(uint8 pin) {
+  #if defined(ESP8266)
+  if (!wifiConnectInProgress) {
+    lastADCvalue = analogRead(A0);
+  }
+  return lastADCvalue;
```

Comment:
this line can lead to wrong readings by error multiplying if last reading was faulty
maybe we can add here counter to check how many readings was during connection in progress just before merge ;)

Reply:
Yep, I was also thinking about something like that.
The reading on ESP32 when reading from ADC2 can also return an error.

I think both ideas are a compromise until we make a way to always accept the reading can fail.